VKapp/VKapp/Controllers/FriendsController.swift

```diff
@@ -121,11 +127,14 @@ final class FriendsController: UITableViewController {
     }
 
     private func loadData() {
+        let realm = try? Realm()
+        print(realm?.configuration.fileURL)
         let dataFromRealm = RealmService.get(Friend.self)
         guard let friendsFromRealm = dataFromRealm else { return }
         addFriendNotificationToken(result: friendsFromRealm)
         if !friendsFromRealm.isEmpty {
             friends = friendsFromRealm
+            photoCacheService = PhotoCacheService(container: tableView)
```

Comment:
вo view did load, в отдельный метод
```suggestion
            photoCacheService = PhotoCacheService(container: tableView)
```